scripts/loans/createLoanWithTerms.js

```diff
@@ -57,7 +58,7 @@ module.exports = async (callback) => {
             requestNonce: nonce,
             amount: amountWithDecimals,
             duration: durationInDays * ONE_DAY,
-            requestTime: Math.round(currentTimestamp / 1000),
```

Comment:
How come this needed to change?

Reply:
Because we changed the Timer JS class in a previous PR. Details here: https://github.com/teller-protocol/teller-protocol-v1/commit/cada826e798b43efa51d365fb2039dbafdbe595e#diff-5814c901e592fb9648cb7436c0538b7e8808f43eadd9cf2d758531dd261e5756